cli/exchange/node.go

```diff
@@ -326,6 +332,8 @@ func NodeRemovePolicy(org, credToUse, node string, force bool) {
 	}
 
 	// remove policy
+	msgPrinter.Printf("Removing Node policy ...")
```

Comment:
"Removing Node policy and re-evaluating all agreements based on just the builtin node policy. Existing agreements might be cancelled and re-negotiated."